demo/src/lib/layout/ComponentTypeAlert.svelte

```diff
@@ -4,23 +4,28 @@
 	import biExclamationTriangleFill from 'bootstrap-icons/icons/exclamation-triangle-fill.svg?raw';
 	import {page} from '$app/stores';
 	import Svg from './Svg.svelte';
+	import {untrack} from 'svelte';
 
 	const regex = /\/(components|services)\/([^/]+)/;
 	const typeIcon: Record<string, string> = {
 		info: biInfoCircleFill,
 		warning: biExclamationTriangleFill,
 	};
 
-	export let componentType: string;
+	let {
+		componentType,
+	}: {
+		componentType: string;
+	} = $props();
 	let alert: Alert;
 
-	$: type = 'info' as AlertProps['type'];
-	$: componentName = $page.url.pathname.match(regex)?.[2];
-	$: {
+	let type = $derived('info' as AlertProps['type']);
+	let componentName = $derived($page.url.pathname.match(regex)?.[2]);
+	$effect(() => {
 		// eslint-disable-next-line @typescript-eslint/no-unused-expressions
 		componentName;
-		alert?.api.open();
-	}
+		untrack(() => alert!.api.open());
```

Comment:
Please add a comment explaining why this workaround of using `untrack` is needed, with the reference to the (existing or new :wink:) Svelte issue that requests the fix of this problem in Svelte.

Reply:
Comment added and issue on our side created !